src/main/java/com/gregtechceu/gtceu/api/fluids/store/FluidStorageKeys.java

```diff
@@ -1,43 +1,59 @@
 package com.gregtechceu.gtceu.api.fluids.store;
 
 import com.gregtechceu.gtceu.GTCEu;
+import com.gregtechceu.gtceu.api.data.chemical.material.Material;
 import com.gregtechceu.gtceu.api.data.chemical.material.info.MaterialIconType;
+import com.gregtechceu.gtceu.api.data.chemical.material.properties.FluidProperty;
 import com.gregtechceu.gtceu.api.data.chemical.material.properties.PropertyKey;
 import com.gregtechceu.gtceu.api.fluids.FluidState;
 
-import java.util.function.UnaryOperator;
+import org.jetbrains.annotations.NotNull;
 
 public final class FluidStorageKeys {
 
     public static final FluidStorageKey LIQUID = new FluidStorageKey(GTCEu.id("liquid"),
             MaterialIconType.liquid,
-            UnaryOperator.identity(),
+            m -> prefixedRegisteredName("liquid_", FluidStorageKeys.LIQUID, m),
             m -> m.hasProperty(PropertyKey.DUST) ? "gtceu.fluid.liquid_generic" : "gtceu.fluid.generic",
             FluidState.LIQUID, 0);
 
     public static final FluidStorageKey GAS = new FluidStorageKey(GTCEu.id("gas"),
             MaterialIconType.gas,
-            UnaryOperator.identity(),
+            m -> m.getName() + "_gas",
```

Comment:
This FOR SURE will delete all existing fluids.
Make a method like the prefixing one but for postfixing